lib/matplotlib/patches.py

```diff
@@ -4075,28 +4106,40 @@ def __init__(self, posA=None, posB=None,
         self._mutation_aspect = mutation_aspect
 
         self.set_dpi_cor(dpi_cor)
-        #self._draw_in_display_coordinate = True
+        # self._draw_in_display_coordinate = True
```

Comment:
are there other mentions of this attribute in the codebase?  if it's just a relic of old code it should just be removed.